ops.go

```diff
@@ -561,6 +561,10 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 	hypervisor.Start(&c.RunConfig)
 }
 
+func getCommandHandler(cmd *cobra.Command, args []string) {
+	api.DownloadPackage(args[0])
```

Comment:
Error handling can be improved. The `DownloadPackage` command, if the argument package does not exists, will download an `xml` response from google storage whose contents claim `access denied` the command does not detect that is hasn't downloaded a package (but a xml file) and succeeds.

Not sure what the plans are for storing packages so I won't attempt to fix this without guidance.

Reply:
https://github.com/nanovms/ops/issues/254 - although i wouldn't spend too much time on it - we'll prob. be moving to a minio at some point in the future

Reply:
Cool. Checking the specified package against the package list would be a quickly implementable check and similar to how apt etc seem to work.